flowise-embed/src/components/Bot.tsx

```diff
@@ -1258,6 +1259,10 @@ export const Bot = (botProps: BotProps & { class?: string }) => {
         if (attempt === maxAttempts) {
           const errorMessage = messageUtils.UNABLE_TO_PROCESS_CHECKLIST_MESSAGE;
 
+          const documentErrors = documentsChecklistError();
```

Comment:
A fim de evitar possíveis sobrescritas por referência, recomendo criar um novo objeto com os valores do `documentsChecklistError` ao invés de fazer uma atribuição de objetos:
```suggestion
          const documentErrors = [...documentsChecklistError()];
```